tests/test_API.py

```diff
@@ -1,13 +1,12 @@
 
 from base import *
-
-
+import vcr
```

Comment:
Yeah, i have used its ruby equivalent a lot. The idea is simple and clever.
Bill Cromie
t:917.215.3576

sent on the move

On Jan 23, 2015 7:34 PM, "Alex" <notifications@github.com> wrote:

> In tests/test_API.py
> <https://github.com/aberke/check-list/pull/6#discussion_r23490825>:
>
> > @@ -1,13 +1,12 @@
> >
> >  from base import *
> > -
> > -
> > +import vcr
>
> cool! The name of this library + it's use case with Twilio (audio) really
> threw me off. This is a neat one!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aberke/check-list/pull/6/files#r23490825>.
>